egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json

```diff
@@ -31571,5 +31571,247 @@
     ],
     "insight": {},
     "_comment": " "
+  },
+  "attendanceSprayOperatorsDistrictPercent" : {
+    "chartName": "DSS_HEALTH_ATTENDANCE_SPRAY_OPERATOR_DISTRICT_PERCENT",
+    "queries": [
+      {
+        "module": "COMMON",
+        "dateRefField": "attendanceLog.time",
+        "requestQueryMap": "{\"province\": \"boundaryHierarchy.province.keyword\", \"district\": \"boundaryHierarchy.district.keyword\"}",
+        "indexName": "attendance-log-index-v1",
+        "aggrQuery": "{\"size\":0,\"aggs\":{\"FILTER\":{\"filter\":{\"bool\":{\"must\":[{\"term\":{\"attendanceLog.type.keyword\":\"EXIT\"}}]}},\"aggs\":{\"Spray Operators Marked Present\":{\"terms\":{\"field\":\"boundaryHierarchy.district.keyword\",\"size\":100},\"aggs\":{\"per_day\":{\"date_histogram\":{\"field\":\"attendanceLog.time\",\"calendar_interval\":\"day\",\"format\":\"yyyy-MM-dd\"},\"aggs\":{\"individuals_with_exit_logs\":{\"terms\":{\"field\":\"attendanceLog.individualId.keyword\",\"size\":10000,\"execution_hint\":\"map\"}},\"total_individuals_with_exit_logs_per_day\":{\"value_count\":{\"field\":\"attendanceLog.individualId.keyword\"}}}},\"Count\":{\"sum_bucket\":{\"buckets_path\":\"per_day>total_individuals_with_exit_logs_per_day\"}}}}}}}}"
+      },
+      {
+        "module": "COMMON",
+        "dateRefField": "Data.createdTime",
+        "requestQueryMap": "{\"province\":\"Data.boundaryHierarchy.province.keyword\",\"district\":\"Data.boundaryHierarchy.district.keyword\"}",
+        "indexName": "project-staff-index-v1",
+        "aggrQuery": "{\"size\":0,\"aggs\":{\"FILTER\":{\"filter\":{\"bool\":{\"must\":[{\"term\":{\"Data.role.keyword\":\"REGISTRAR\"}},{\"term\":{\"Data.projectTypeId.keyword\":\"192a20d1-0edd-4108-925a-f37bf544d6c4\"}}]}},\"aggs\":{\"Total spray operators\":{\"terms\":{\"field\":\"Data.boundaryHierarchy.district.keyword\",\"size\":500},\"aggs\":{\"Count\":{\"value_count\":{\"field\":\"Data.id.keyword\"}}}}}}}}"
```

Comment:
why is the field projectTypeId has a hardcoded filter?

Reply:
because all campaigns have unique projectTypeId. and if we have more than one campaign's data projectTypeId helps distinguish between then

Reply:
This is a case for all of the apis not only for this, we shouldn't be using a hardcoded value here. How will someone use this config in other environment? so remove this filter

Reply:
are you suggesting we send projectTypeId for this chart as a filter through UI then?